.release-please-manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  ".": "2.12.0",
-  "packages/api-explorer": "0.9.41",
-  "packages/code-editor": "0.1.26",
-  "packages/extension-api-explorer": "22.20.0",
-  "packages/extension-sdk": "22.20.0",
-  "packages/extension-sdk-react": "22.20.0",
-  "packages/extension-utils": "0.1.18",
-  "packages/hackathon": "22.20.0",
-  "packages/run-it": "0.9.41",
-  "packages/sdk": "22.20.0",
-  "packages/sdk-codegen": "21.7.3",
-  "packages/sdk-codegen-scripts": "21.4.7",
+  ".": "2.13.0",
+  "packages/api-explorer": "0.9.42",
+  "packages/code-editor": "0.1.27",
+  "packages/extension-api-explorer": "22.21.0",
+  "packages/extension-sdk": "22.20.1",
+  "packages/extension-sdk-react": "22.20.1",
+  "packages/extension-utils": "0.1.19",
+  "packages/hackathon": "22.21.0",
```

Comment:
Hmm. I think we need to manually update the manifest to `"release-as": ""` I'll get that PR up